numba/core/withcontexts.py

```diff
@@ -464,3 +466,70 @@ def _mutate_with_block_callee(blocks, blk_start, blk_end, inputs, outputs):
         block=ir.Block(scope=scope, loc=loc),
         outputs=outputs,
     )
+
+class _ParallelChunksize(WithContext):
```

Comment:
In the interests of keeping parfors specific code together, should this be moved into a module under `numba.parfors` ?

Reply:
I'm torn.  I can see arguments on both sides.  I'll leave the decision up to you.